easybuild/easyconfigs/g/GCCcore/GCCcore-6.1.0.eb

```diff
@@ -0,0 +1,57 @@
+easyblock = 'EB_GCC'
+
+name = 'GCCcore'
+version = '6.1.0'
+
+homepage = 'http://gcc.gnu.org/'
+description = """The GNU Compiler Collection includes front ends for C, C++, Objective-C, Fortran, Java, and Ada,
+ as well as libraries for these languages (libstdc++, libgcj,...)."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+mpfr_version = '3.1.4'
+
+source_urls = [
+    'http://ftpmirror.gnu.org/gmp',  # idem for GMP
+    'http://ftpmirror.gnu.org/mpfr',  # idem for MPFR
+    'http://www.multiprecision.org/mpc/download',  # MPC official
+    'ftp://gcc.gnu.org/pub/gcc/infrastructure/',  # GCC dependencies
+    'http://gcc.cybermirror.org/infrastructure/',  # HTTP mirror for GCC dependencies
+    'http://ftp.heanet.ie/mirrors/gnu/%(namelower)s/%(namelower)s-%(version)s/', # Alternative for GCC
```

Comment:
No worries, when running the build before (without the mirror link) it complained about not being able to find and download gcc-6.1.0.tar.bz2 so I've added another alternative URL in its place which I don't think is a mirror.